### PR TITLE
Mannequins

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
you can now craft mannequins using 25 wood or plastic
they work with the mob stripping system letting you put items on em!
image
image

## Why It's Good For The Game
fun stuff for CARGO FASHION showcases.

## Changelog
:cl:
add: You can now craft mannequins with wood or plastic.
/:cl:

### CODE REVIEW

**Comment by Serxule** [comment]
make it so you can change the underwear on it (socks, underwear, undershirt)

**Review by Imaginos16 — APPROVED** [review]
LOVE IT

**Comment by Shadowflame909** [comment]
Manuel's Fashion Shop DLC just dropped

**Comment by Farquaar** [comment]
Would love to see plastic versions as well, so I can put creepy white plastic mannequins in my clown lair.

**Comment by calliophis-maculiceps** [comment]
fikou is literally the only guy adding good content to the game

**Comment by SuperSlayer0** [comment]
Make it a special interaction with wizard bolts of animation like statues have

**Comment by Shadowflame909** [comment]
> Make it have a special interaction with wizard bolts of animation like statues have

It'd be funny if they were stamina damage version of the statues. Since they're made out of wood and not marble.

**Comment by Y0SH1M4S73R** [comment]
> Make it have a special interaction with wizard bolts of animation like statues have

why not make them into humans with a hostile AI?

File: code/game/objects/structures/mannequin.dm [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+	. = ..()
+	if(!.)
+		return FALSE
+	var/obj/structure/mannequin/mannequin_source = source
+	if(!istype(mannequin_source))
+		return FALSE
+	var/obj/item/clothing/suit = mannequin_source.worn_items["[ITEM_SLOT_OCLOTHING]"]
+	if(istype(suit) && is_type_in_list(src, suit.allowed))
+		return TRUE
+	to_chat(user, span_warning("[equipping]2 won't fit!"))
```
Comment (Mothblocks) [inline code/game/objects/structures/mannequin.dm:219]: 2?

**Comment by unit0016** [comment]
Givin' me inspiration to rework the null body mob over here. :eyes:

File: code/game/objects/structures/mannequin.dm (hunk 1 above)
Comment (Fikou) [reply code/game/objects/structures/mannequin.dm:219]: accident

**Comment by Fikou** [comment]
> Would love to see plastic versions as well, so I can put creepy white plastic mannequins in my clown lair.

sure

**Comment by Wallemations** [comment]
Demanding to know if they're unanchored so I can put them on a conveyor belt and have them walk the catwalk

**Comment by Fikou** [comment]
> Demanding to know if they're unanchored so I can put them on a conveyor belt and have them walk the catwalk

they start unanchored, you can use a wrench to anchor

**Comment by VladinXXV** [comment]
> > Make it have a special interaction with wizard bolts of animation like statues have
> 
> why not make them into humans with a hostile AI?

real boy

**Review by tralezab — APPROVED** [review]
fine...

**Comment by tralezab** [comment]
*grumbling about mat datums going unused*

**Pull request merged**